server/postgresql/psql1.js

```diff
@@ -0,0 +1,46 @@
+
+const { Client } = require('pg');
+const client = new Client({
+  user: 'username',
+  host: 'localhost',
+  database: 'sdc',
+  password: 'password123',
```

Comment:
may be best practice to place pwd into gitignore

Reply:
Thanks for catching that, I'll add that into my next commit.